tests/components/ring/conftest.py

```diff
@@ -27,20 +27,15 @@ def mock_setup_entry() -> Generator[AsyncMock, None, None]:
 @pytest.fixture
 def mock_ring_auth():
     """Mock ring_doorbell.Auth."""
-    with patch("ring_doorbell.Auth", autospec=True) as mock_ring_auth:
+    with patch(
+        "homeassistant.components.ring.config_flow.Auth", autospec=True
+    ) as mock_ring_auth:
         mock_ring_auth.return_value.fetch_token.return_value = {
```

Comment:
This currently doesn't mock the Auth used in `__init__.py` is that correct?

Reply:
Yes that's correct, for now the one in `__init__.py` still relies on the `requests_mock` and doesn't use this fixture yet.